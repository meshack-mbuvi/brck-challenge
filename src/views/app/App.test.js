import React from "react";
import App from ".";

import { Provider } from "react-redux";

import store from "../../redux/store";

import Enzyme, { shallow } from "enzyme";

import Adapter from "enzyme-adapter-react-16";
import expect from "expect";
import sinon from "sinon";

Enzyme.configure({ adapter: new Adapter() });

describe("Renders <App />", () => {
  it("renders without crashing", () => {
    const getRestaurantsFromCSV = sinon.spy();
    const wrapper = shallow(
      <Provider store={store}>
        <App />
      </Provider>
    ).debug();

    expect(wrapper).toMatchSnapshot();
    expect(getRestaurantsFromCSV.called);
  });
});
