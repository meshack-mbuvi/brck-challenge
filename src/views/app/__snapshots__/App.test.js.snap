// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <App /> renders without crashing 1`] = `
"<ContextProvider value={{...}}>
  <Connect(index) />
</ContextProvider>"
`;
